service/controller/controllercontext/status_types.go

```diff
@@ -103,6 +104,59 @@ type ContextStatusTenantClusterTCCPVPC struct {
 	PeeringConnectionID string
 }
 
+type ContextStatusTenantClusterTCCPN struct {
+	ASG                      ContextStatusTenantClusterTCCPNASG
+	AvailabilityZones        []ContextStatusTenantClusterTCCPNAvailabilityZone
+	DockerVolumeResourceName string
+	EtcdVolumeSnapshotID     string
+	Image                    string
+	IsTransitioning          bool
+	ResourceName             string
+	Type                     string
+	VPC                      ContextStatusTenantClusterTCCPNVPC
+}
```

Comment:
Let'S talk about this. I feel like all we need is the snapshot ID and the stack outputs. All the other information are already part of the context and it does not appear to be necessary to duplicate this information within the context structure.